invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/uploads_items/MobileUploadsItem.js

```diff
@@ -0,0 +1,133 @@
+import _truncate from "lodash/truncate";
+import React from "react";
+import { Button, Icon, Item, Label } from "semantic-ui-react";
+import { SearchItemCreators } from "../../utils";
+import PropTypes from "prop-types";
+
+export const MobileUploadsItem = ({
+  result,
+  editRecord,
+  statuses,
+  access,
+  uiMetadata,
+}) => {
+  const { accessStatusId, accessStatus, accessStatusIcon } = access;
+  const {
+    descriptionStripped,
+    title,
+    creators,
+    subjects,
+    publicationDate,
+    resourceType,
+    createdDate,
+    version,
+    isPublished,
+    viewLink,
+  } = uiMetadata;
+
+  const icon = isPublished ? (
+    <Icon name="check" className="positive" />
+  ) : (
+    <Icon name="upload" className="negative" />
+  );
+
+  return (
+    <Item key={result.id} className="deposits-list-item mb-20 mobile only flex">
+      <Item.Content className="centered">
+        <Item.Extra className="labels-actions">
+          {result.status in statuses && result.status !== "published" && (
+            <Label size="tiny" className={statuses[result.status].color}>
+              {statuses[result.status].title}
+            </Label>
+          )}
+          <Label size="tiny" className="primary">
+            {publicationDate} ({version})
+          </Label>
+          <Label size="tiny" className="neutral">
+            {resourceType}
+          </Label>
+          <Label size="tiny" className={`access-status ${accessStatusId}`}>
+            <i className={`icon ${accessStatusIcon}`} />
+            {accessStatus}
+          </Label>
+        </Item.Extra>
+        <Item.Header as="h2">
+          <a href={viewLink} className="truncate-lines-2">
+            {icon}
+            {title}
+          </a>
+        </Item.Header>
+        <Item.Meta>
+          <div className="creatibutors">
+            <SearchItemCreators creators={creators} className="ml-auto mr-auto" />
+          </div>
+        </Item.Meta>
+        <Item.Description>
+          {_truncate(descriptionStripped, {
+            length: 100,
+          })}
+        </Item.Description>
+        <Item.Extra>
+          <Item.Extra>
+            {subjects.map((subject) => (
+              <Label key={subject.title_l10n} size="tiny">
+                {subject.title_l10n}
+              </Label>
+            ))}
+            <div>
+              {}
+              <small>
+                {createdDate ? (
+                  <>
+                    {i18next.t("Uploaded on")} <span>{createdDate}</span>
+                  </>
+                ) : (
+                  i18next.t("No creation date found.")
+                )}
+              </small>
+            </div>
+          </Item.Extra>
+        </Item.Extra>
+        <Item.Extra>
+          <Button
+            compact
+            size="small"
+            floated="right"
+            onClick={() => editRecord()}
+            className="fluid-responsive"
+          >
+            <Icon name="edit" />
```

Comment:
maybe there is a difference in how it looks but otherwise you can also pass the `icon` prop to the button 😄 . Same as below